hack/generate/changelog/util/fragment_test.go

```diff
@@ -370,3 +370,36 @@ func TestGitPullRequestNumberGetter_parsePRNumber(t *testing.T) {
 		})
 	}
 }
+
+func Test_gitPullRequestNumberGetter_getCommitMessage(t *testing.T) {
+	type args struct {
+		filename string
+	}
```

Comment:
Nit: any reason for the extra struct wrapper for `args`? Could we put `filename` directly in the test struct instead of `args`?

Reply:
good point, fixed it.